src/TimeService.Business/Commands/WorkTime/FindWorkTimesCommand.cs

```diff
@@ -0,0 +1,63 @@
+﻿using LT.DigitalOffice.Kernel.AccessValidatorEngine.Interfaces;
+using LT.DigitalOffice.Kernel.Broker;
+using LT.DigitalOffice.Kernel.Exceptions.Models;
+using LT.DigitalOffice.Kernel.Extensions;
+using LT.DigitalOffice.TimeService.Business.Commands.WorkTime.Interfaces;
+using LT.DigitalOffice.TimeService.Data.Filters;
+using LT.DigitalOffice.TimeService.Data.Interfaces;
+using LT.DigitalOffice.TimeService.Mappers.Models.Interfaces;
+using LT.DigitalOffice.TimeService.Models.Dto.Models;
+using MassTransit;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace LT.DigitalOffice.TimeService.Business.Commands.WorkTime
+{
+    public class FindWorkTimesCommand : IFindWorkTimesCommand
+    {
+        private readonly IWorkTimeInfoMapper _mapper;
+        private readonly IWorkTimeRepository _repository;
+        private readonly IAccessValidator _accessValidator;
+        private readonly IHttpContextAccessor _httpContextAccessor;
+
+        public FindWorkTimesCommand(
+            IWorkTimeInfoMapper mapper,
+            IWorkTimeRepository repository,
+            IAccessValidator accessValidator,
+            IHttpContextAccessor httpContextAccessor)
+        {
+            _mapper = mapper;
+            _repository = repository;
+            _accessValidator = accessValidator;
+            _httpContextAccessor = httpContextAccessor;
+        }
+
+        public WorkTimesResponse Execute(FindWorkTimesFilter filter, int skipCount, int takeCount)
+        {
+            if (filter == null)
+            {
+                throw new ArgumentNullException(nameof(filter));
+            }
+
+            var userId = _httpContextAccessor.HttpContext.GetUserId();
+
+            if (filter.UserId.HasValue && filter.UserId != userId && !_accessValidator.IsAdmin())
+            {
+                throw new ForbiddenException("Not enough rights.");
+            }
+
+            int totalCount;
```

Comment:
put it in line 54

Reply:
It used on 58

Reply:
and what is your problem? you can write out int totalCount and use it later